test/components/dragBoxLayerTests.ts

```diff
@@ -317,7 +272,6 @@ describe("Interactive Components", () => {
       });
```

Comment:
this is not in the test description 
can you either update the description or move this into another test?